antlr-kotlin-runtime/build.gradle.kts

```diff
@@ -1,113 +1,57 @@
-plugins {
-    id("org.jetbrains.kotlin.multiplatform")
-}
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
-repositories {
-    mavenCentral()
-    mavenLocal()
+plugins {
+  alias(libs.plugins.kotlin.multiplatform)
 }
 
-apply(plugin = "maven-publish")
-
 kotlin {
-    jvm()
-    js(BOTH) {
-        browser {
-        }
-        nodejs {
-        }
-    }
+  @OptIn(ExperimentalKotlinGradlePluginApi::class)
+  compilerOptions {
+    apiVersion = KotlinVersion.KOTLIN_1_9
+    languageVersion = KotlinVersion.KOTLIN_1_9
+    freeCompilerArgs.add("-Xexpect-actual-classes")
+  }
 
-    ios("ios") {
-        binaries {
-            staticLib()
-        }
-    }
-    linuxArm64 {
-        binaries {
-            staticLib()
-        }
+  jvm {
+    compilations.configureEach {
+      compilerOptions.configure {
+        jvmTarget.set(JvmTarget.JVM_1_8)
+        freeCompilerArgs.addAll(
+          "-Xjvm-default=all",
+          // TODO: verify the performance impact of null assertions
```

Comment:
Perhaps we can remove these lines and open an issue instead

Reply:
Yup those are just reminders. Will get rid of them before merging.  
I was discussing about assertions on Kotlin's Slack, and will open a YouTrack issue to get the same flags for the JS target.